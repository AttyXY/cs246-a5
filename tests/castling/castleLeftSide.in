game human human
move d2 d4
move d7 d5
move d1 d3
move d8 d6
move c1 d2
move c8 d7
move b1 a3
move b8 a6
move e1 c1
move e8 c8