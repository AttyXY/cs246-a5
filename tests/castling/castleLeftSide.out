WELCOME TO CHESS! PLEASE ENTER ONE OF THE FOLLOWING COMMANDS TO BEGIN:
- setup
- game [player1] [player2]

STARTING GAME!

8  rnbqkbnr
7  pppppppp
6  --------
5  --------
4  --------
3  --------
2  PPPPPPPP
1  RNBQKBNR
   abcdefgh

8  rnbqkbnr
7  pppppppp
6  --------
5  --------
4  ---P----
3  --------
2  PPP-PPPP
1  RNBQKBNR
   abcdefgh

8  rnbqkbnr
7  ppp-pppp
6  --------
5  ---p----
4  ---P----
3  --------
2  PPP-PPPP
1  RNBQKBNR
   abcdefgh

8  rnbqkbnr
7  ppp-pppp
6  --------
5  ---p----
4  ---P----
3  ---Q----
2  PPP-PPPP
1  RNB-KBNR
   abcdefgh

8  rnb-kbnr
7  ppp-pppp
6  ---q----
5  ---p----
4  ---P----
3  ---Q----
2  PPP-PPPP
1  RNB-KBNR
   abcdefgh

8  rnb-kbnr
7  ppp-pppp
6  ---q----
5  ---p----
4  ---P----
3  ---Q----
2  PPPBPPPP
1  RN--KBNR
   abcdefgh

8  rn--kbnr
7  pppbpppp
6  ---q----
5  ---p----
4  ---P----
3  ---Q----
2  PPPBPPPP
1  RN--KBNR
   abcdefgh

8  rn--kbnr
7  pppbpppp
6  ---q----
5  ---p----
4  ---P----
3  N--Q----
2  PPPBPPPP
1  R---KBNR
   abcdefgh

8  r---kbnr
7  pppbpppp
6  n--q----
5  ---p----
4  ---P----
3  N--Q----
2  PPPBPPPP
1  R---KBNR
   abcdefgh

8  r---kbnr
7  pppbpppp
6  n--q----
5  ---p----
4  ---P----
3  N--Q----
2  PPPBPPPP
1  --KR-BNR
   abcdefgh

8  --kr-bnr
7  pppbpppp
6  n--q----
5  ---p----
4  ---P----
3  N--Q----
2  PPPBPPPP
1  --KR-BNR
   abcdefgh

Final Score:
White: 0
Black: 0
THANKS FOR PLAYING CHESS!
